### ISSUE TITLE
Generating an invoice only after payment is done

### ISSUE THREAD

**Issue by Floflinou** [Sylius/Sylius#10686]
Hello,

My problem is that the invoices are generated when an order is placed, I would like to have it generated when the order is paid. I've been trying to mess around in the config but I couldn't figure it out.

Thanks in advance !

**Comment by Zales0123** [comment]
Duplicate of https://github.com/Sylius/InvoicingPlugin/issues/155 🖖
